Tasks/ManualValidationV0/task.json

```diff
@@ -0,0 +1,60 @@
+{
+    "id": "2003AD3A-104A-451A-9238-60474AB294E6",
+    "name": "ManualValidation",
+    "friendlyName": "Manual validation",
+    "description": "Pause deployment and wait for manual validation",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/manual-validation",
+    "helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?linkid=870234)",
+    "category": "Deploy",
+    "visibility": [
+        "Release"
+    ],
+    "author": "Microsoft Corporation",
+    "runsOn": [
+        "Server"
+    ],
+    "version": {
+        "Major": 0,
+        "Minor": 0,
+        "Patch": 1
+    },
+    "inputs": [
+        {
+            "name": "validators",
+            "type": "identities",
+            "label": "Manual Validators",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Only users added in validators list can act on a manual validation."
+        },
+        {
+            "name": "instructions",
+            "type": "multiLine",
+            "label": "Instructions",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "These instructions will be shown to the user for resuming or rejecting the manual intervention. Based on these instructions the user will take an informed decision about this manual intervention.",
+            "properties": {
+                "resizable": "true",
+                "rows": "10",
+                "maxLength": "4000"
+            }
+        },
+        {
+            "name": "onTimeout",
+            "type": "radio",
+            "label": "On timeout",
+            "defaultValue": "reject",
+            "required": false,
+            "options": {
+                "reject": "Reject",
+                "resume": "Resume"
+            },
+            "helpMarkDown": "Reject or resume this manual intervention automatically after it is pending for the specified timeout or 60 days, whichever is earlier."
```

Comment:
Can we please confirm if its 60 days (Also can we add the default option here as in what will be the default behavior as this input is non-mandatory)